docs/teamwork.md

```diff
@@ -0,0 +1,19 @@
+# Working with us
```

Comment:
Another subtle philosophical thing: maybe this could be something more like "How we collaborate" or "How we work" instead of "Working with us"?

To me, the "us" distinction feels like it creates more distance between the core team and new or outside contributors